gtfs/spec/en/reference.md

```diff
@@ -82,6 +82,7 @@ This specification defines the following files:
 |  [transfers.txt](#transferstxt)  | Optional | Rules for making connections at transfer points between routes. |
 |  [pathways.txt](#pathwaystxt)  | Optional | Pathways linking together locations within stations. |
 |  [levels.txt](#levelstxt)  | Optional | Levels within stations. |
+|  [translations.txt](#translationstxt)  | Optional | Translations of dataset values. |
```

Comment:
`Translations of customer-facing dataset values.`

Reply:
Thanks! I update it in https://github.com/google/transit/pull/218/commits/513725f0874e901247621f2f617e7718e55a5609.